index.html

```diff
@@ -1,11 +1,32 @@
+<!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.0/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="style.css">
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title> Apex UI</title>
 </head>
 <body>
-    
+    <header class="navbar">
+        <div id="logo">
+            <a href="index.html">
+                <span><img class="logo-icon" src="./assets/logo/apex-legends-symbol-white.png" alt="A"></span><span class="name">pex UI</span></a>
+        </div>
+        <div class="Home">
+            <a href="https://github.com/bossmonkey98"><i class="bi bi-github"></i></a>
+            <a href="https://www.linkedin.com/in/arun-kumar-0b95911aa/"><i class="bi bi-linkedin"></i></a>
+        </div>
+    </header>
+    <div class="content-container">
+        <p>
+           Now design your website easily and faster with <strong style="font-family: Georgia, 'Times New Roman', Times, serif; color:cyan;">Apex UI</strong>
+        </p>
+        <div>
+            <a href="docs.html"><button class="btn-1">Documentation</button></a>
+            <button class="btn-2">Installation</button>
```

Comment:
Give a meaningful class name it don't look btn-2 or 3. Provide like btn-explore or primary or secondary or anything else.